<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tab_dataformat</class>
 <widget class="QWidget" name="tab_dataformat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_26">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Format:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cmb_dataformat_type">
     <item>
      <property name="text">
       <string>int8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>int16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>int32</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>uint8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>uint16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>uint32</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>float</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>double</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_24">
     <property name="text">
      <string>Prefix:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="input_dataformat_prefix">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_23">
     <property name="text">
      <string>Suffix:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="input_dataformat_suffix">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_25">
     <property name="text">
      <string>Delimiter:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="input_dataformat_delimiter">
     <property name="text">
      <string>,</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="chk_datafilter_enable">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Filter:</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QTreeWidget" name="tree_datafilter">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="indentation">
      <number>0</number>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerMinimumSectionSize">
      <number>0</number>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
